board/xilinx/zynq/board.c

```diff
@@ -44,7 +44,7 @@ static xilinx_desc fpga100 = XILINX_XC7Z100_DESC(0x100);
 #ifndef CONFIG_TPL_BUILD
 static struct {
   uint32_t hardware;
-  uint32_t serial_number;
+  char uuid[17];
```

Comment:
The 17 was a mistake since I switched between using the mfg id and the uuid. We can add the manufacturer serial number back to this struct, I wasn't sure how it would be used going forward.